node-packages/commons/src/tasks.ts

```diff
@@ -1148,7 +1148,15 @@ export const createMiscTask = async function(taskData: any) {
           // Handle setting up the configuration for a restic restoration task
           const restoreName = `restore-${R.slice(0, 7, taskData.data.backup.backupId)}`;
           // generate the restore CRD
-          const restoreConf = restoreConfig(restoreName, taskData.data.backup.backupId, makeSafe(taskData.data.project.name))
+
+          let baasBucketName = ""
+          for (let variable of osResult.envVariables) {
+            if (variable.name == "LAGOON_BAAS_BUCKET_NAME") {
+              baasBucketName = variable.value
+            }
+          }
+          // Parse out the baasBucketName for any migrated projects
+          const restoreConf = restoreConfig(restoreName, taskData.data.backup.backupId, makeSafe(taskData.data.project.name), makeSafe(baasBucketName))
```

Comment:
do we need to run `makeSafe()` on the `baasBucketName`? this will change the `baasBucketName` with replacing an `_` to a `-` which could cause the restore to fail as the backup system maybe actually used a `_`

Reply:
We don't _need_ to run that here, no. I'll update accordingly.